packages/hash/api/src/auth/ory-kratos.ts

```diff
@@ -1,19 +1,20 @@
 import { Configuration, Identity } from "@ory/client";
 import {
-  V0alpha2Api as OpenSourceV0alpha2Api,
-  AdminCreateIdentityBody,
+  CreateIdentityBody,
+  FrontendApi,
+  IdentityApi,
 } from "@ory/kratos-client";
 import { getRequiredEnv } from "../util";
 
 const publicUrl = getRequiredEnv("ORY_KRATOS_PUBLIC_URL");
 
-export const publicKratosSdk = new OpenSourceV0alpha2Api(
+export const publicKratosSdk = new FrontendApi(
```

Comment:
Similarly we could rename this to something like `kratosFrontendApi`

Reply:
Thanks, applied in 1ca939b057484c10cf54f7660758dc1ce4e0cb19